cms/config/sync/core-store.plugin_content_manager_configuration_content_types##api##layer.layer.json

```diff
@@ -105,7 +105,7 @@
       },
       "interaction_config": {
         "edit": {
-          "label": "Interaction Config",
```

Comment:
I think here the config changing the labels to more user friendly ones gets overridden, but its a minor thing that would be easier to fix separately